client/landing/gutenboarding/onboarding-block/design-selector/index.tsx

```diff
@@ -106,9 +106,7 @@ const DesignSelector: FunctionComponent = () => {
 								onClick={ ( e: MouseEvent< HTMLDivElement > ) => {
 									window.scrollTo( 0, 0 );
 									setCp( e.currentTarget.offsetTop );
-									setSelectedDesign( currentTemplate =>
-										currentTemplate?.slug === design?.slug ? undefined : design
-									);
+									setSelectedDesign( selectedDesign?.slug === design?.slug ? undefined : design );
```

Comment:
`design` isn't nullableish, is it?

```suggestion
									setSelectedDesign( selectedDesign?.slug === design.slug ? undefined : design );
```